src/test/lit-html_test.ts

```diff
@@ -124,6 +124,30 @@ describe('grimlock', () => {
         `);
       });
 
+      it('reflecting property expressions', () => {
+        expect(
+          convertModule(
+            'test.ts',
+            js`
+                import {html} from 'lit-html';
+
+                /**
+                 * @soyCompatible
+                 */
+                export const t = () => {
+                  return html\`<input .value=\${"foo"} .className=\${"bar"}>\`
```

Comment:
Let's add some TODOs for additional test cases:

* `class` attribute binding and `.className` property binding on same element (error?)
* `<input>` with `.value` and `.checked` bindings (do nothing special?)
* Any element with `.id`, `.tabIndex`, etc.
* `.onclick`, etc. event bindings (hopefully they warn already, but we should give a nice suggestion to use event bindings)